File: Java/Car.java
package Java;

class Car {
    private Integer id;
    private String license;
    private Account driver;
    protected Integer passenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDataCar() {
        if(passenger != null)
            System.out.println("license: " + license + " Driver: " + driver.name + " passenger: " + passenger);
    }
    //*     ID
    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }
    //*     License
    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }
    //*     Driver
    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }
    //*     Passenger
    public Integer getPassenger() {
        return passenger;
    }

    public void setPassenger(Integer passenger) {
        if (passenger == 4) {
            this.passenger = passenger;
        } else {
            System.out.println("Necesitas asignar 4 pasajeros");
        }
    }
}

File: PHP/Car.php
<?php
require_once('account.php');
class Car {
    //* Declaration of atributes
    private $id;
    private $license;
    private $driver;
    protected $passenger;

    public function __construct($license, $driver){
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCar() {
        if ($this->passenger != null){
            echo "
            -\nLicencia: $this->license
            Driver: {$this->driver->name}
            Número de pasajeros: $this->passenger \n";
        }
    }

    public function getPassenger() {
        return $this->passenger;
    }

    public function setPassenger($passenger) {

        if ($passenger == 4) {
            $this->passenger = $passenger;
        }
        else {
            echo "Necesitas asignar 4 pasajeros";
        }

    }
}
?>
